src/lib/Hydra.pm

```diff
@@ -48,6 +48,14 @@ __PACKAGE__->config(
             file($ENV{'HYDRA_LDAP_CONFIG'})
         ) : undef
     },
+    'Plugin::ConfigLoader' => {
+        driver => {
+            'General' => {
```

Comment:
Pretty good! Could you add a comment in this file, around this line, to also update https://github.com/NixOS/hydra/blob/2f0dc294afc512bd3db1c3476b8d46954214af53/src/lib/Hydra/Helper/Nix.pm#L45-L48 if these General options change, and also edit https://github.com/NixOS/hydra/blob/2f0dc294afc512bd3db1c3476b8d46954214af53/src/lib/Hydra/Helper/Nix.pm#L45-L48 to say to edit this files' options to match? This way future changes don't get out of sync again.

Reply:
PS I don't suppose you would be up for writing a bit of documentation about this?

`foreman start manual` will run a live-reloading docs webserver. Maybe a snippet in https://github.com/NixOS/hydra/blob/master/doc/manual/src/installation.md, or a new configuration.md file.

Note if you add a new file (I sort of prefer this...) you'll need to add it to the SUMMARY: https://github.com/NixOS/hydra/blob/master/doc/manual/src/SUMMARY.md

Reply:
Done. I'm segfaulting on whether factoring out two lines is worth a new file.

Reply:
... or we can export it from `Nix.pm`. I'll look into it in the morning. I'm perled-out for the day.